src/databricks/labs/ucx/assessment/crawlers.py

```diff
@@ -1,8 +1,12 @@
 import base64
+import json
 import logging
 import re
 
+from databricks.sdk import WorkspaceClient
 from databricks.sdk.errors import NotFound
+from databricks.sdk.service import compute
+from databricks.sdk.service.compute import ClusterDetails, Policy
```

Comment:
line 9 is redundant because you've imported the whole `compute` package. use one style of imports, don't mix

Reply:
fixed